bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieRequestProcessor.java

```diff
@@ -412,6 +415,7 @@ private OrderedExecutor createExecutor(
                     .numThreads(numThreads)
                     .name(nameFormat)
                     .traceTaskExecution(serverCfg.getEnableTaskExecutionStats())
+                    .preserveMdcForTaskExecution(serverCfg.getPreserveMdcForTaskExecution())
```

Comment:
use local cached value  instead of getting from conf again?

Reply:
I'll leave this the way it is.
createExecutor() is called in the constructor so changing it to the local value creates expectations for the order of the value initialization.  
Performance-wise it is a noop since it is only used in the constructor.